spec/system/profile/index_spec.rb

```diff
@@ -159,7 +160,8 @@
     # valid_until set to 1 year from now
     expect(page).to have_content("Mindeslaufzeit: bis #{I18n.l(1.year.from_now, format: '%m/%Y')}")
 
-    Timecop.travel(6.months.from_now + 1.minute)
+    Timecop.travel(6.months.from_now + 3.hours.from_now)
+
```

Comment:
I put this is a separate PR, but after re-running the test 4 times, I decided to include it here because the flakiness has become untenable